### ISSUE TITLE
It is suggested to add the accuracy to test the model.

### ISSUE THREAD

**Issue by 0x0004** [facebookresearch/fastText#1027]
Only Recall and F1 are inconvenient and not intuitive...

**Comment by 0x0004** [comment]
@cpuhrsch

**Comment by 0x0004** [comment]
@ola13

**Comment by 0x0004** [comment]
@EdouardGrave

**Comment by 0x0004** [comment]
@Celebio

**Comment by MrRaghav** [comment]
Hello, I just wanted to know if there is any plan to add this feature or I should use sklearn.metrics for the time being?
